docs/miscellaneous-options/performance.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Performance
+weight: 13
+---
+
+
+This feature helps to reduce memory when using [data urls](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URLs).
```

Comment:
Could you better explain what this feature does. Make it a paragraph of human friendly text